src/Subscriber/ActionSubscriber.php

```diff
@@ -338,8 +339,8 @@ public function handleEvent(ActionEvent $event): ActionEvent
             $this->io->newLine();
         }
 
-        $this->logger->info("$totalActions/$totalActions -- Finished looping all Actions listening to \"$listeningToThrow\"");
-
+        // Commented out this log, to avoid log creation overload. Only add back for debug reasons.
+        // $this->logger->info("$totalActions/$totalActions -- Finished looping all Actions listening to \"$listeningToThrow\"");
         return $event;
 
     }//end handleEvent()
```

Comment:
I'm not sure if this is what they want, it could actually be that Rotterdam is complaining about the style-logs, as debug and info are not stored in online environments at this time.

Reply:
good point